### PR TITLE
no results font color and weight (bug 1127515)

### PR DESCRIPTION
**Pull request by spasovski**

### CODE REVIEW

File: src/media/css/lib/colors.styl [hunk 1]
```diff
@@ -59,6 +59,7 @@ $footer-border = #e5e4e5;
 $page-background = $greyscale-light-grey;
 $tile-border = $greyscale-light-grey;
 $tile-title = #222;
+$no-results-text = #929292;
```
Comment (ngokevin) [inline src/media/css/lib/colors.styl]: y not palette?

File: src/media/css/search.styl [hunk 2]
```diff
@@ -1,6 +1,8 @@
 @import 'lib';
 
 .no-results {
+    font-weight: 500;
+    color: $no-results-text;
     padding: 50px;
```
Comment (ngokevin) [inline src/media/css/search.styl]: abc

File: src/media/css/search.styl [hunk 3]
```diff
@@ -1,6 +1,8 @@
 @import 'lib';
 
 .no-results {
+    font-weight: 500;
```
Comment (ngokevin) [inline src/media/css/search.styl]: Not a fan... :(

Comment (spasovski) [inline src/media/css/search.styl]: Oh I agree...it was requested that it's 100% the same as before (I kind of ignored the padding) thus the weirdo old color also.

Comment (ngokevin) [inline src/media/css/search.styl]: Can we choose the closest color? I think it won't be discernable

**Comment by ngokevin** [comment]
i'm r+ if we can use a palette color...or get confirmation that we are absolutely against using a palette color

**Comment by spasovski** [comment]
yep picking one now...was looking at something else in the meanwhile

File: src/media/css/search.styl [hunk 4]
```diff
@@ -1,6 +1,8 @@
 @import 'lib';
 
 .no-results {
+    color: $action-neutral;
```
Comment (spasovski) [inline src/media/css/search.styl]: damn those neutrals!

**Pull request merged**